wp-includes/widgets/class-wp-widget-media-video.php

```diff
@@ -59,16 +59,10 @@ public function get_instance_schema() {
 		$schema = array_merge(
 			parent::get_instance_schema(),
 			array(
-				'poster' => array(
-					'type' => 'string',
-					'default' => '',
-					'format' => 'uri',
-					'description' => __( 'URL to the poster frame' ),
-				),
 				'preload' => array(
 					'type' => 'string',
 					'enum' => array( 'none', 'auto', 'metadata' ),
-					'default' => 'none',
+					'default' => 'auto',
```

Comment:
Was just curious about this change to `auto` is that the typical default in the editor video flow?